src/database/seeders/20190801105352-role.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('Roles', [
+      {
+        id: '45429837-ed2c-435d-bc22-ad9c5dbe3782',
+        type: 'requester',
+        createdAt: new Date(),
```

Comment:
You don't need to use the New date function because you migration generate instant date for you

Reply:
OK. I'll alert when I make the changes.

Reply:
okay do rebase when you through

Reply:
Done